docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -0,0 +1,139 @@
+# Setting up monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Deploy Prometheus Operator](#deploy-prometheus-operator)
+  * [Configure Prometheus Operator](#configure-prometheus-operator)
+  * [Install Prometheus Operator](#install-prometheus-operator)
+* [Accessing Prometheus Operator sub-components](#accessing-prometheus-operator-sub-components)
+  * [Accessing Prometheus](#accessing-prometheus)
+  * [Accessing Alertmanager](#accessing-alertmanager)
+  * [Accessing Grafana](#accessing-grafana)
+    * [Using port forward](#using-port-forward)
+    * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide provides installation steps of monitoring stack using prometheus-operator component and ways to access the sub-components like Prometheus, Alertmanager and Grafana.
+
+## Prerequisites
+
+- A Lokomotive cluster is accessible via `kubectl` deployed on a supported provider.
+
+<!---
+TODO: Once we have tutorials on how to deploy and configure rook or openebs, then point the following to those tutorials.
+-->
+
+- Deploy a storage provider component (rook and rook-ceph or openebs-operator and openebs-storage-class) with default storage class that can cater to the [PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) requests made by prometheus-operator components Alertmanager and Prometheus.
+
+- [MetalLB](https://metallb.universe.tf/) deployed on the cluster if you wish to [expose Grafana using Ingress](#using-ingress).
+
+  **NOTE**: Required only for the bare metal and Packet providers.
+
+  Installation instructions for [MetalLB](./ingress-with-contour-metallb.md) component.
```

Comment:
I find it weird that we break the list to insert the note and the link to the MetalLB doc. I'd rather put the note *after* the list (and mark it with `>` for style).
I'd consider linking to the MetalLB component from the MetalLB bullet itself instead of linking to the MetalLB website.